### DISCUSSION TITLE
[Discussion] No clear concept about which version should be installed

### DISCUSSION THREAD

**Discussion by Cyberavater** [ElderDrivers/EdXposed#576] [Q&A] [answered]
In the Magisk module description (or anywhere for that matters), I couldn't find any clear documentation about the difference between AHFA or SandHook. Which version should be installed based on what, Why should one install one instead of the other one.

Please consider updating the documentation regarding this.

**Comment by eebssk1** [comment]
That's actually only a matter of performance.You can try all of them then choose the one that have best performance while not causing issues.Sandhook should provide best performance but it may cause issues according to some users' report.

**Comment by ChristophSchmidpeter** [comment, marked as answer]
@eebssk1 I think putting that info that in general SandHook has better performance, but YAHFA better stability would be a great guideline to put into the documentation.
